### PR TITLE
cli: Provide aliases for kata-* subcommands and options

### PR DESCRIPTION
**Pull request by knittl**
This PR solves some of the problems described in #1011. It's not perfect, but a start :)

### CODE REVIEW

File: src/runtime/cli/kata-check.go [hunk 1]
```diff
@@ -306,8 +306,9 @@ func genericHostIsVMContainerCapable(details vmContainerCapableDetails) error {
 }
 
 var kataCheckCLICommand = cli.Command{
-	Name:  checkCmd,
-	Usage: "tests if system can run " + project,
+	Name:    "check",
+	Aliases: []string{"kata-check"},
```
Comment (bergwolf) [inline src/runtime/cli/kata-check.go:310]: cool! Didn't know that we can have aliases here.

**Review by bergwolf — APPROVED** [review]
Thanks!

File: src/runtime/cli/kata-check.go (hunk 1 above)
Comment (knittl) [reply src/runtime/cli/kata-check.go:310]: I even tried for the options `--kata-config` and `--kata-show-default-config-paths`, but Go wouldn't let me, even though the documentation for `cli.StringFlag`/`cli.BoolFlag` suggests that aliases can be set there too. Getting the options to work can be a follow-up MR

**Review by Tim-Zhang — APPROVED** [review]

**Comment by Tim-Zhang** [comment]
@knittl Thanks, lgtm

**Review by jodh-intel — APPROVED** [review]
Thanks very much @knittl !

lgtm

**Pull request merged**